index.md

```diff
@@ -1,3 +1,11 @@
+---
+title: Petition to remove the Board of the Free Software Foundation and Richard M. Stallman from the GNU Project
+description: We, the undersigned, are calling for the removal of the entire Board of the Free Software Foundation, for its support of toxic and abusive behavior by Richard M. Stallman.
```

Comment:
I suggest we quote the document rather than using a summary (as this language is not already present in the repo). Same feedback for the graphic.

Reply:
I think that is a good point. Given that the title itself should be relatively short, do you have a suggestion for that? For the description I will take a verbatim section from the introduction of the letter.

Reply:
"An open letter to remove Richard M. Stallman from all leadership positions" ?

Reply:
Alrighty, changes made in most recent commit:
- Change title to your suggestion
- Change description to "We, the undersigned…" from the letter
- Change text on share image to list demands in the form of verbatim text from letter